Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision 44a58aaba3a8fd9e8cd589ad2457d1c747b1ff59)
+++ b/main.py	(date 1675862143727)
@@ -13,7 +13,7 @@
         self.ctext = ''.join([self.__ALPHABET[self.caesar_code_i(i)] for i in self.text])
 
     def caesar_decode_i(self, i):
-        return self.__ALPHABET.index(i) - self.__KEY % len(self.__ALPHABET)
+        return (self.__ALPHABET.index(i) - self.__KEY) % len(self.__ALPHABET)
 
     def caesar_decode(self):
         self.text = ''.join([self.__ALPHABET[self.caesar_decode_i(i)] for i in self.ctext])
